R/scale_ggsurvfit.R

```diff
@@ -34,8 +34,20 @@
 #' # specify additional scales
 #' ggsurvfit +
 #'   scale_ggsurvfit(x_scales = list(breaks = seq(0, 8, by = 2)))
+#'
+#' # Specical case: in the risk table, large numbers (with more than 4 digits) may not
+#' # be shown completely, with some digits truncated outside the plot region.
+#' # To remedy this, consider the following example code for a simulated large-size survival dataset:
+#' # Here, adjust the size in `add_risktable(size = 2.5)` (as an example) can decrease the font size of the risk table
```

Comment:
The reason people see this issue is because the default padding when using `scale_ggsurvit()` is too restrictive, so the best "fix" is going to live here in this function rather than reducing the font size of the risk table.

The proposed fix is going to be a change to the expand scale argument in this function.

Reply:
Thank you @ddsjoberg a lot for your suggestion, I've already addressed this issue and inside the new commit.